painttiming.bs

```diff
@@ -6,7 +6,10 @@ Level: 1
 ED: https://w3c.github.io/paint-timing/
 TR: https://www.w3.org/TR/paint-timing/
 Status: ED
-Editor: Shubhie Panicker, Google https://google.com, panicker@google.com
+Editor: Nicolás Peña Moreno, Google https://google.com, npm@chromium.org
+        Noam Rosenthal
```

Comment:
@noamr I don't know if you want to provide a company and/or contact email here.

Reply:
Noam Rosenthal, invited expert <noam@webkit.org>

(Is it customary to write invited expert in these? That’s my affiliation here..., leave out if it’s not the custom)

Reply:
I think it can help clarify your affiliation, so would be good to add it.